src/blueapi/worker/reworker.py

```diff
@@ -193,11 +215,11 @@ def _report_status(
         warnings = self._warnings
         if self._current is not None:
             task_status = TaskStatus(
-                task_name=self._current.name,
+                task_name=self._current.task_id,
```

Comment:
Is there a way to avoid having `task_name` and `task_id` as two names for the same thing?

Reply:
Yes, was being conservative because that would change the external API, but we currently  have no users so it's fine...